src/coreclr/vm/methodtablebuilder.h

```diff
@@ -126,10 +126,12 @@ class MethodTableBuilder
         METHOD_TYPE_FCALL   = 1,
         METHOD_TYPE_NDIRECT = 2,
         METHOD_TYPE_EEIMPL  = 3,
```

Comment:
I'm going to see about removing this `enum` entirely and defer to `MethodClassification` everywhere.